app/coffee/HttpController.coffee

```diff
@@ -198,4 +215,15 @@ module.exports = HttpController =
 			else
 				res.send project_ids
 
-
+	flushQueuedProjects: (req, res, next = (error) ->) ->
+		options = 
+			limit : req.query.limit || 1000
+			timeout: 5 * 60 * 1000
+			min_delete_age: req.query.min_delete_age || 5 * 60 * 1000
+		res.send 204
```

Comment:
By making this run in the background, I think we can't use the kubernetes cron control features to avoid running multiple jobs at once. I think that might be good or bad. If we force it to flush one project at a time, there's a risk that it won't be fast enough to keep up, and allowing multiple concurrent jobs would let it catch up, albeit in a somewhat ad hoc way. WDYT?

Reply:
Ah I wasn't aware that kubernetes cron jobs had concurrency control, that's useful so it's better to run it in the foreground. Now fixed.